lecture/views.py

```
@@ -2,48 +2,104 @@
-        return redirect(f'{reverse("lecture_home")}?slug={slug}')
-    return render(request, "lecture/create_form.html", {"form": form, 'form_type': 'lecture'})
+@require_http_methods(["GET", "POST"])
+def create_lecture(request: HttpRequest) -> HttpResponse:
+    if request.method == "POST":
+        form = LectureForm(request.POST)  # Pass the user if needed
+
+        if form.is_valid():
+            saved_lecture = form.save()
+            messages.success(request, "Lecture created successfully!")
+            return redirect("lecture_home", slug=saved_lecture.slug)  # Redirect to a relevant page
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)